extensions/amp-story-shopping/0.1/amp-story-shopping-tag.css

```diff
@@ -51,38 +52,178 @@ amp-story-shopping-tag {
+  }
+  100% {
+    transform: scale(1);
+  }
+}
+
+@keyframes pulse-dot-after {
+  0% {
+    animation-timing-function: cubic-bezier(0.76, 0, 0.24, 1);
+    transform: scale(1);
+  }
+  40% {
+    transform: scale(.8);
+  }
+  90% {
+    transform: scale(1);
+  }
+  100% {
+    transform: scale(1);
+  }
 }
 
 .i-amphtml-amp-story-shopping-tag-pill {
+  --i-amphtml-pill-icon-size: 24px !important;
+  --i-amphtml-pill-padding-inline-start: 6px !important;
+  --i-amphtml-pill-transfrom-origin: calc(var(--i-amphtml-pill-padding-inline-start) + var(--i-amphtml-pill-icon-size) * .5) !important;
   display: flex !important;
   align-items: center !important;
-  border-radius: 100px !important;
+  border-radius: 18px !important;
   height: 100% !important;
+  overflow: hidden !important;
   background-color: var(--i-amphtml-shopping-tag-bg-color) !important;
-  backdrop-filter: var(--i-amphtml-shopping-tag-backdrop-filter) !important;
-  padding: 4px 0px !important;
-  margin-inline-end: 4px !important;
+  padding-inline-start: var(--i-amphtml-pill-padding-inline-start) !important;
+  padding-inline-end: 12px !important;
+  transform-origin: var(--i-amphtml-pill-transfrom-origin);
+}
+
+[active] .i-amphtml-amp-story-shopping-tag-pill {
+  animation: animate-in-sequence-pill var(--i-amphtml-shopping-tag-animation-sequence-duration) forwards,
+    animate-out-sequence-pill var(--i-amphtml-shopping-tag-animation-sequence-duration) var(--i-amphtml-shopping-tag-animate-out-delay) forwards !important;
+
+  /* Values that need to change to support RTL. */
+  --clip-path-start-circle-supports-rtl: circle(calc(var(--i-amphtml-pill-icon-size) * .5) at calc(var(--i-amphtml-pill-padding-inline-start) + var(--i-amphtml-pill-icon-size) * .5)) !important;
+  --clip-path-end-circle-supports-rtl: circle(calc(var(--i-amphtml-pill-icon-size) * .5 + var(--i-amphtml-pill-padding-inline-start)) at calc(var(--i-amphtml-pill-padding-inline-start) + var(--i-amphtml-pill-icon-size) * .5)) !important;
+  --clip-path-inset-offset-supports-rtl: inset(0 calc(100% - 36px) 0 0 round 30px) !important;
+}
+
+[active][dir=rtl] .i-amphtml-amp-story-shopping-tag-pill {
+  --clip-path-start-circle-supports-rtl: circle(calc(var(--i-amphtml-pill-icon-size) * .5) at calc(100% - var(--i-amphtml-pill-padding-inline-start) - var(--i-amphtml-pill-icon-size) * .5)) !important;
+  --clip-path-end-circle-supports-rtl: circle(calc(var(--i-amphtml-pill-icon-size) * .5 + var(--i-amphtml-pill-padding-inline-start)) at calc(100% - var(--i-amphtml-pill-padding-inline-start) - var(--i-amphtml-pill-icon-size) * .5)) !important;
+  --clip-path-inset-offset-supports-rtl: inset(0 0 0 calc(100% - 36px) round 30px) !important;
+
+  /* Transform orign from center of icon (right side) in RTL. */
+  transform-origin: calc(100% - var(--i-amphtml-pill-transfrom-origin)) !important;
+}
+
+@keyframes animate-in-sequence-pill {
+  0% {
+    animation-timing-function: cubic-bezier(.85, 0, .15, 1);
```

Comment:
Just curious, does this override the other timing function for this part of the animation only?